deltaspike/parent/pom.xml

```diff
@@ -49,7 +49,8 @@
     <properties>
         <java.version>1.6</java.version>
         <owb.version>1.2.7</owb.version>
-        <weld.version>1.1.28.Final</weld.version>
+        <!-- Weld profiles (in parent/code/pom.xml) override this version -->
+        <weld.version>1.1.9.Final</weld.version>
```

Comment:
This is based on a Gerhard's comment. He claimed that the idea is to list the minimal supported version. Although I strongly dislike dislike this approach, I put it here.
However if you look into Weld1 profile, you can see it is overriden by current version (1.1.33.Final). So this version declaration is naught but a mere information :)